test/codeu/chat/server/storage/StorageTest.java

```diff
@@ -2,42 +2,38 @@
 
```

Comment:
General comments/tips about writing tests:

1. Please structure your tests in this fashion:

- Create all the objects which are the inputs for your test case.
- Pass the inputs to your method under test and get back the result.
- Assert the result against the expected values. The expected values should be created manually whenever possible and should not be in any way derived from the results of calling the method under test.

2. Try to store very minimal state (or better none at all if possible) within your test class (i.e. `StorageTest` here). Any instance variables you declare should be initialized only within the `@Before` method.

3. Each test should start from a clean state and should not have any dependencies on things that may or may not have been initialized by other test methods. For example, a bug in another test method should not cause this test method to fail. Invoking `reset()` here is one such brittleness.

4. Avoid if conditions and for loops in tests as much as possible. If you need a loop, make its size defined explicitly instead of deriving from another state created during the test. The goal is to have reproducible tests which always perform the same test independent of the methods under test or any dependent classes.